source/jormungandr/jormungandr/scenarios/new_default.py

```diff
@@ -793,6 +793,8 @@ def merge_responses(responses, debug):
                 continue
             merged_response.impacts.extend([i])
 
+    journey_filter.remove_excess_tickets_or_ticket_links_with_journeys_to_be_deleted(merged_response)
```

Comment:
Why not filtering at [line 772](https://github.com/CanalTP/navitia/blob/9a1d6268cab30668ff1bcccdbc20b631f2d5aedd/source/jormungandr/jormungandr/scenarios/new_default.py#L772)?
```
tickets_to_add = set(t for j in r.journeys for t in j.fare.ticket_id)
```
only adding ticket from non deleted journeys should be enough no?

Reply:
Good point, I can handle it to probably safe process loop call.